src/index.html

```diff
@@ -5,8 +5,432 @@
+          <img 
+            src="images/logo.png" 
+            alt="moyoLogo"
+          >
+        </a>
+      </div>
+
+      <nav class="nav">
+        <ul class="nav__list">
+          <li class="nav__item">
+            <a 
+              href="#apple"
+              class="nav__link is-active nav__link--is-active">Apple
+            </a>
+
+          </li>
+          <li class="nav__item">
+            <a 
+              href="#samsung"
+              class="nav__link">Samsung
+            </a>
+          </li>
+
+          <li class="nav__item">
+            <a 
+              href="#smartphones"
+              class="nav__link">Smartphones
+            </a>
+          </li>
+
+          <li class="nav__item">
+            <a 
+              href="#laptAndComp" 
+              data-qa="nav-hover"
+              class="nav__link">Laptops & Computers
+            </a>
+          </li>
+
+          <li class="nav__item">
+            <a 
+              href="#gadgets"
+              class="nav__link">Gadgets
+            </a>
+          </li>
+
+          <li class="nav__item">
+            <a 
+              href="#tablets"
+              class="nav__link">Tablets
+            </a>
+          </li>
+
+          <li class="nav__item">
+            <a 
+              href="#home"
+              class="nav__link">Photo
+            </a>
+          </li>
+
+          <li class="nav__item">
+            <a 
+              href="#home"
+              class="nav__link">Video
+            </a>
+          </li>
+          
+        </ul>
+      </nav>
+    </header>
+
+    <main class="page__catalog catalog">
+      <div 
+        class="catalog__card card"
+        data-qa="card"
+      >
+      
+        <img 
+          src="images/imac.jpeg"
+          alt="Apple iMac27"
+          class="card__image"
+        >
+      
+        <div class="card__title-container">
+          <h1 class="card__title-text">
+            APPLE A1419 iMac 27" Retina 
+            5K Monoblock (MNED2UA/A)
+          </h1>
+      
+          <p class="card__title-catalog-code">
+            Product code: 195434
+          </p>
+        </div>
+
+        <div class="card__reviews">
+          <div class="card__stars stars stars--4">
+            <div class="stars__star"></div>
+            <div class="stars__star"></div>
+            <div class="stars__star"></div>
+            <div class="stars__star"></div>
+            <div class="stars__star"></div>
+          </div>
+          <p class="card__reviews-text">Reviews: 5</p>
+        </div>
+
+        <div class="card__price-container">
+          <p class="card__price-text">Price:</p>
+          <p class="card__price-value">$2,199</p>
+        </div>
+
+        <a 
+          href="#home"
+          class="card__button"
+          data-qa="card-hover"
+        >buy
+        </a>
+      </div>
+      <div 
+        class="catalog__card card" 
+      >
+      
+        <img 
+          src="images/imac.jpeg" 
```

Comment:
```suggestion
      </div>
      
      <div class="catalog__card card">
        <img 
          src="images/imac.jpeg" 
```
add empty lines only between siblings